plugin/src/main/kotlin/org/gradle/github/dependency/extractor/ForceDependencyResolutionPlugin.kt

```diff
@@ -0,0 +1,26 @@
+package org.gradle.github.dependency.extractor
+
+import org.gradle.api.Plugin
+import org.gradle.api.invocation.Gradle
+
+@Suppress("unused")
+class ForceDependencyResolutionPlugin : Plugin<Gradle> {
+    override fun apply(gradle: Gradle) {
+        gradle.projectsEvaluated {
+            val resolveAllDeps = gradle.rootProject.tasks.create("dependencyExtractor_resolveBuildDependencies")
```

Comment:
The line where this plugin is applied says: `// Apply the dependency resolver to all builds`
Then this first line creates the `dependencyExtractor_resolveBuildDependencies` task on the `rootProject` every time this plugin is applied. Doesn't that mean that this task is created redundantly?

Reply:
If I understand correctly, this `Gradle` plugin will be applied to each _build_ (not _project_), and each build has a single rootProject.
But I should be using `tasks.register` not `tasks.create`....

Reply:
`register` may not be Gradle 5 compatible. Not sure, you'll have to check